.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
^^ Please add `.vscode/` inside your development machine's `.gitignore` file. But I'm guessing since you haven't changed the variables in this file, it's better to just delete it.